docs/api_walkthrough.rst

```diff
@@ -87,8 +87,9 @@ Successfully opening a channel will return the following information::
         "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
         "token_address": "0x9aBa529db3FF2D8409A1da4C9eB148879b046700",
         "balance": 1337,
-        "state": "open",
-        "settle_timeout": 600
+        "state": "opened",
+        "settle_timeout": 600,
+        "reveal_timeout": 10
```

Comment:
There is no need to specify reveal timeout. It's kind of an esoteric option which we would not like to advertise too much as being available so it was left out on purpose. Can you remove it?

Reply:
Sure